packages/aws-cdk-lib/aws-dynamodb/lib/table-v2.ts

```diff
@@ -483,6 +498,8 @@ export class TableV2 extends TableBaseV2 {
 
   protected readonly region: string;
 
+  protected readonly tags: TagManager;
```

Comment:
I see that you chose [the ITaggable  approach ](https://github.com/aws/aws-cdk/blob/be70c822a1608cb43a4b4b17fc0430f3352797c6/packages/aws-cdk-lib/core/lib/tag-manager.ts#L236). Why not to add it to the interfaces that this class implements to be easier for readers to know that this class is taggable.